packages/eslint-config/index.js

```diff
@@ -1,3 +1,52 @@
 module.exports = {
-  extends: 'airbnb-base',
-};
+    extends: 'airbnb-base',
+    rules: {
+        'arrow-parens': ['error', 'as-needed'],
+        'comma-dangle': ['error', 'only-multiline'],
+        'eol-last': 'off',
+        /*
```

Comment:
I would prefer if we had new lines between all the rules, such that it looked like this (cleaner, better separation between rules):

```json
        'max-classes-per-file': ['error', 1],
        
        /*
            This is the default value which is defined here to note that there may be exceptions
            where it is appropriate to disable this rule with a comment. Awaits in loops are
            expected to be mistakes more often than not. However, there may be exceptions for
            unique situations.
        */
        'no-await-in-loop': 'error',
        
        'no-console': 'error',
        
        /*
            This is the default value which is defined here to note that there may be exceptions,
            like array iterators, where it is appropriate to disable this rule with a comment.
        */
        'no-loop-func': 'error',
        
        /*
            This is the default value which is defined here to note that this rule defends against
            prototype pollution vulnerabilities that exploitable with XSS.
        */
        'no-prototype-builtins': 'error',
```

Reply:
That is fine with me.